docs/design/features/framework-version-resolution.md

```diff
@@ -30,7 +30,7 @@ In the `.runtimeconfig.json` these values are defined like this:
 ```
 
 #### Framework name
-Each framework reference identifies the framework by its name.  
+Each framework reference identifies the framework by its name.
 Framework names are case sensitive (since they're used as folder names even on Linux systems).
```

Comment:
This was in separate lines, but now will become in the same line. Do you want them to be in separate lines again?

Reply:
I think it is fine to be on the same line. I would change the raw text to have it on the same line as well.